src/03-calendar/events.apib

```diff
@@ -49,7 +49,15 @@ Get a list of calendar events.
                 + description (string)
                 + starts_at: `2016-02-04T16:00:00+00:00` (string)
                 + ends_at: `2016-02-04T18:00:00+00:00` (string)
-                + location (string)
+                + location (string)`
```

Comment:
Q: does this single quote have a purpose?

Reply:
Nope, removed it

 :ok: by @Jdww on 07/02/2018 14:31:48